src/pages/Resources/Resources.js

```diff
@@ -57,9 +61,23 @@ function Resources() {
             <strong> {count}</strong> results.
           </Typography>
         )}
+        {results && renderResults()}
+        <br />
+        {!searchValue && (
+          <Pagination
+            variant="outlined"
+            shape="rounded"
+            size="large"
+            count={Math.floor(count / 10)}
```

Comment:
small nitpick: I would move computations like these out of the rendering of components and before the return

Reply:
Good suggestion -- thanks @turretd!